apps/website-25/.env.production.template

```diff
@@ -0,0 +1,3 @@
+# This file will be picked up by the deploy script for production environment.
+NEXT_PUBLIC_GA_MEASUREMENT_ID=G-C5L967C9M1
+NEXT_PUBLIC_POSTHOG_KEY=phc_NVqRwH6xZ0MiP9I5a9tx3l9zsTWBvMI1YEbvbCOXDcI
```

Comment:
Should these be different to the staging keys?

Reply:
Hi @Will-Howard, thanks for looking out 👍🏻. I have not created the actually staging `KEY`s (yet). I noted this down as a `To-do` in the `Rollout`-section of the `README`.

Reply:
Maybe it's fine to use the same keys across dev and staging, at least for now? If we do really want to keep these isolated it'd probably be fine to split them out later.

Reply:
Good call, @domdomegg! I'll update our `.env.local.template` to use `staging` keys.